index.html

```diff
@@ -81,18 +81,39 @@ <h2>Make the marinade</h2>
     <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="Marinade">Combine Ingredients for the chicken marinade and let chicken marinade for at least 30 minutes<br></li>
   </ol>
   <br>
-  <h2 class="mar">Cook chicken with sauce</h2>
-  <ol>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="1">Cook chicken in an oiled pan until brown on both sides/fully cooked<br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="2">Add 1 tbsp butter and 1 tbsp oil to pan. Fry 1 large chopped onion until it browns <br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="3">Add 1 1/2 tbsp of minced garlic and 1 tbsp of minced ginger. Stir until fragrant<br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="4">Add 1 tbsp of ground coriander, 1 1/2 tbsp of cumin and 1 1/2 tbsp of garam masala. Stir till fragrant<br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="5">Add 14 oz of crushed tomatoes, 1 tsp of chili powder and 1 1/4 tsp of salt. Simmer for 10-15 minutes
-      until color deepens with occasional stirring<br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="6">Blend sauce in a mixer until smooth<br></li>
-    <li><input type="checkbox" onload="return writeTo(this)" name="check_list" value="7">Add 1 cup of heavy cream, 1 tbsp of sugar and 1/2 tsp of kasoori methi. Let cook for an additional 10
-      minutes<br></li>
-  </ol>
+  <h2 class="mar">Cook chicken with sauce test</h2>
+  <div class="chickenMarinade">
+      <ol class="chickenMarinadeList">
```

Comment:
Notice here the approach I use of applying specific classes to elements I want to change so I know that the styles I apply to these elements will affect only these elements. I similar approach will work so you can apply `float: left` to the nav bar elements without applying it to all list elements.

Also note how I reformatted this. I always use linters e.g. prettier, or the Airbnb linter. Even if you don't add a linter to all the code you write right now, it would be helpful to start getting familiar with the rules so you can write code that makes sense to any engineer.